lib/generators/rails_template18f/oscal/templates/bin/trestle.tt

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+command="bash"
+if [ "$1" != "" ]; then
+    command=$1
+fi
+
+oscal_location="$(dirname $(realpath $0))/../doc/compliance/oscal"
```

Comment:
suggestion (non-blocking): Handle files/dirs with spaces.

If $0 includes a space things will get sad. This fixes it at the cost of double quote mind melting.

```suggestion
oscal_location="$(dirname "$(realpath "$0")")/../doc/compliance/oscal"
```

Reply:
my brain can't comprehend that that double quoting works properly, but my computer seems to agree with you